app/error.tsx

```diff
@@ -17,17 +20,21 @@ export default function Error({ error, reset }: ErrorProps) {
   }, [error]);
 
   return (
-    <div className="flex flex-col m-auto">
-      <h2 className="flex flex-col m-auto">Oops! Something went wrong!</h2>
-      <button
-        className="border-2 flex flex-col m-auto"
-        onClick={
-          // This button triggers the reset function, which will try to re-render the Error Boundary's contents without triggering a full page reload.
-          () => reset()
-        }
-      >
-        Try again
-      </button>
+    <div>
+      <h2 className="mt-12 text-3xl text-left font-semibold">Oops!</h2>
+      <p className="mt-4 text-left">Something went wrong!</p>
+      <div className="mt-12 mb-12" aria-label='Error page' aria-live='assertive'>
+        <Player
+        autoplay={true}
+        loop={true}
+        className="w-[350px] h-[350px]"
+        src={ErrorAnimation}
+        >
+        </Player>
+      </div>
+      <p className="mb-48">
```

Comment:
Maybe move `click` outside of the hyperlink text to clarify `here` is the word to interact with for navigation?

Also we can add some styling, such as bold text, change color when hover, and add an underline for email etc. to make the `here` and `email` stand out so the user is clear where to click.

```
      <p className="mb-48">
        Please click<a href="#" onClick={() => reset()}className="font-bold hover:text-red-500"> here</a> to refresh the page or contact us at <a href="mailto:info@linkta.org" className="underline hover:text-red-500">info@linkta.org</a>
      </p>
```